bioimageio/core/prediction_pipeline/_processing.py

```diff
@@ -209,7 +209,7 @@ def apply(self, tensor: xr.DataArray) -> xr.DataArray:
         v_lower = get_stat(ref_name, Percentile(self.min_percentile, axes=axes))
         v_upper = get_stat(ref_name, Percentile(self.max_percentile, axes=axes))
 
-        return ensure_dtype((tensor - v_lower) / v_upper, dtype="float32")
+        return ensure_dtype((tensor - v_lower) / (v_upper - v_lower), dtype="float32")
```

Comment:
IMO this update makes a lot of sense to actually map the max percentile to 1...
I'm inclined to view the current implementation as buggy and would like to just fix/update it.
The spec is not specific: https://github.com/bioimage-io/spec-bioimage-io/blob/main/supported_formats_and_operations.md#preprocessing
 @esgomezm @tomburke-rse what are your implementations/expectations. Can we just change it?

Reply:
In imagej-modelzoo, the percentile normalization already does this as far as I can see:
https://github.com/imagej/imagej-modelzoo/blob/master/src/main/java/net/imagej/modelzoo/plugin/transformation/preprocessing/NormalizePercentilePreprocessing.java

Reply:
We also divide using v_lower: https://github.com/deepimagej/imagej-macros/blob/648caa867f6ccb459649d4d3799efa1e2e0c5204/bioimage.io/per_sample_scale_range.ijm#L92

```
	//normalization
	diff = ma-mi+1e-20; // add epsilon to avoid 0-divisions
	run("32-bit");
	run("Subtract...", "value="+mi+" stack");
	run("Divide...", "value="+diff+" stack");
```

Reply:
Ok, then let's update it!